src/mol-model/structure/structure/util/superposition-sifts-mapping.ts

```diff
@@ -85,9 +91,26 @@ function getPositionTables(index: IndexEntry[], pivot: number, other: number, N:
         }
     }
 
+    if (traceOnly) {
+        xs.x = xs.x.slice(0, o);
+        xs.y = xs.y.slice(0, o);
+        xs.z = xs.z.slice(0, o);
+        ys.x = ys.x.slice(0, o);
+        ys.y = ys.y.slice(0, o);
+        ys.z = ys.z.slice(0, o);
+    }
     return [xs, ys];
 }
 
+function findTraceAtom(unit: Unit.Atomic, start: ElementIndex, end: ElementIndex): [Unit.Atomic, ElementIndex, ElementIndex] | undefined {
+    for (let i = start; i < end; i++) {
+        const l = unit.model.atomicHierarchy.atoms.label_atom_id.value(i);
```

Comment:
A small nitpick, but maybe move ``unit.model.atomicHierarchy.atoms.label_atom_id`` out of the loop.